content/en/docs/instrumentation/java/exporters.md

```diff
@@ -171,14 +219,240 @@ public class DiceApplication {
 }
 ```
 
-To see the traces exported quickly, you can run Jaeger with OTLP enabled in a
-docker container:
+## Console
+
+To debug your instrumentation or see the values locally in development, you can
+use exporters writing telemetry data to the console (stdout).
+
+If you followed the
+[Getting Started](/docs/instrumentation/java/getting-started/) or
+[Manual Instrumentation](/docs/instrumentation/java/manual/) guides, you already
+have the console exporter installed.
+
+The `LoggingSpanExporter`, the `LoggingMetricExporter` and the
+`SystemOutLogRecordExporter` are included in the
+`opentelemetry-exporter-logging` artifact.
```

Comment:
Might be good to mirror the comment above about how to specify the logging exporter when using autoconfigure:

> Note, that in the case of exporting via OTLP you do not need to set
`OTEL_TRACES_EXPORTER`, `OTEL_METRICS_EXPORTER` and `OTEL_LOGS_EXPORTER` since
`otlp` is their default value

The value which selects the logging exports is `logging`, or `logging-otlp` for a version which logs the OTLP json encoding of the data (though `logging-otlp` requires the `opentelemetry-exporter-logging-otlp` artifact).

Reply:
~~~suggestion
`opentelemetry-exporter-logging` artifact.

If you use
[SDK auto-configuration](/docs/instrumentation/java/manual/#automatic-configuration)
all you need to do is update your environment variables:
```shell
env OTEL_TRACES_EXPORTER=logging OTEL_METRICS_EXPORTER=logging OTEL_LOGS_EXPORTER=logging  java -jar ./build/libs/java-simple.jar
```
~~~

Like this? The `logging-otlp` is an interesting corner case, I am wondering if it should live here OR be better used in the OTLP section since it is helpful there for debugging?

Reply:
👍

> The logging-otlp is an interesting corner case, I am wondering if it should live here OR be better used in the OTLP section since it is helpful there for debugging?

Yeah probably worth omitting logging-otlp for now.